x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/data/BlockAccountingTests.java

```diff
@@ -45,7 +45,7 @@ public void testBooleanVector() {
         Vector emptyPlusOne = new BooleanArrayVector(new boolean[] { randomBoolean() }, 1);
         assertThat(emptyPlusOne.ramBytesUsed(), is(alignObjectSize(empty.ramBytesUsed() + 1)));
 
-        boolean[] randomData = new boolean[randomIntBetween(1, 1024)];
+        boolean[] randomData = new boolean[randomIntBetween(2, 1024)];
```

Comment:
subsequent filtering is on position 1, so we must have at least two positions in our data.